src/dashboard/cd-dashboard-events.vue

```diff
@@ -3,11 +3,27 @@
     <div class="cd-dashboard-events">
       <div class="cd-dashboard-events__content">
         <h1 class="cd-dashboard-events__header">{{ $t('Hey {name}, here\'s what\'s most important...', { name: loggedInUser.firstName }) }}</h1>
-        <div class="cd-dashboard-events__list">
+        <div class="cd-dashboard-events__list" v-if="events">
           <upcoming-event v-for="event in events" :key="event.id" :event="event" :dojo="dojos[event.dojoId]"></upcoming-event>
+          <div class="cd-dashboard-events__cta" v-if="events && events[0].id">
+            <router-link class="cd-dashboard-events__cta-link" :to="{ name: 'MyTickets' }" v-ga-track-click="'your_events'">{{ $t('Your Events') }}</router-link>
+          </div>
         </div>
-        <div class="cd-dashboard-events__cta">
-          <router-link class="cd-dashboard-events__cta-link" :to="{ name: 'MyTickets' }" v-ga-track-click="'your_events'">{{ $t('Your Events') }}</router-link>
+        <div v-else-if="ticketingAdmins.length > 0">
+          <p v-if="!usesTicketing && maxDojoAge < 1" class="cd-dashboard-events__hint">
+            {{ $t('Create your first event so attendees can book and you can easily see who\'s attending.') }}
```

Comment:
We'd also want this to link to event creation. It would be especially useful for new Dojos who haven't yet created their first event.